### PR TITLE
Add rest manu pages, add main page

### PR DESCRIPTION
**Pull request by rafalgawlik**

### CODE REVIEW

File: config/urls.py [hunk 1]
```diff
@@ -7,13 +7,16 @@
...
+from poznaj.views import MainView
+
 router = routers.DefaultRouter()
 router.register(r'images', ImagesViewSet)
 router.register(r'points', PointsViewSet)
 router.register(r'stories', StoriesViewSet)
 
 urlpatterns = [
     url(settings.ADMIN_URL, admin.site.urls),
+    url(r'^', MainView.as_view()),
```
Comment (krzysztofzuraw) [inline config/urls.py]: Please change it to `^$` as tests are broken

File: poznaj/templates/base.html [hunk 2]
```diff
@@ -5,6 +5,53 @@
 <head>
   <meta http-equiv="Content-Type" content="text/html;charset=UTF-8">
   <title>{% block title %}{% endblock %}</title>
+
+
+  <style>
+      body {
```
Comment (krzysztofzuraw) [inline poznaj/templates/base.html]: Can you add those as additional CSS file instead of having them here

File: poznaj/templates/main.html [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <meta name="description" content="">
+    <meta name="author" content="">
+
+    <title>PoznajApp | Poznaj swoje miasto</title>
+
+    <!-- Site CSS -->
+    <link rel="stylesheet" href="{% static 'vendor/bootstrap/css/bootstrap.min.css' %}">
```
Comment (krzysztofzuraw) [inline poznaj/templates/main.html]: Can we serve these via some kind of CDN?

Comment (rafalgawlik) [reply poznaj/templates/main.html]: Can. But what is CDN? :D

File: poznaj/templates/base.html (hunk 2 above)
Comment (rafalgawlik) [reply poznaj/templates/base.html]: 👍 ok

File: config/urls.py (hunk 1 above)
Comment (rafalgawlik) [reply config/urls.py]: 👍  ok

File: poznaj/templates/main.html (hunk 3 above)
Comment (krzysztofzuraw) [reply poznaj/templates/main.html]: CDN: https://pl.wikipedia.org/wiki/Content_Delivery_Network. Instead us they host js & css

**Comment by krzysztofzuraw** [comment]
Please fix lint errors:
```
/home/travis/build/KlubJagiellonski/poznaj-app-backend/poznaj/views.py:3:1: E302 expected 2 blank lines, found 1
/home/travis/build/KlubJagiellonski/poznaj-app-backend/poznaj/mobile/views.py:3:1: E302 expected 2 blank lines, found 1
/home/travis/build/KlubJagiellonski/poznaj-app-backend/poznaj/mobile/views.py:6:1: E302 expected 2 blank lines, found 1
/home/travis/build/KlubJagiellonski/poznaj-app-backend/poznaj/mobile/views.py:9:1: E302 expected 2 blank lines, found 1
/home/travis/build/KlubJagiellonski/poznaj-app-backend/poznaj/mobile/views.py:12:1: E302 expected 2 blank lines, found 1
```

**Pull request closed without merging**